activerecord/lib/active_record/transactions.rb

```diff
@@ -461,7 +461,7 @@ def transaction_include_any_action?(actions) #:nodoc:
           when :create
             transaction_record_state(:new_record)
           when :destroy
-            destroyed?
+            @_deleted_rows_in_db ||= false
```

Comment:
The assignment is needed to avoid: `warning: instance variable @_deleted_rows_in_db not initialized`

Is there a better way to do this?